NuCivic/dkan
# Fix colorizer pagination css

https://github.com/NuCivic/dkan/issues/609

## Description

Fix colorizer css to include pagination tabs.

## QA Steps
1. Change the color settings at `admin/appearance/colorizer` and save
2. Navigate to `admin/appearance/fontyourface`
3. Confirm that the colors of the sub-nav tabs match the new color scheme

![appearance___dkan](https://user-images.githubusercontent.com/314172/28144514-86014160-6731-11e7-8a46-6ffd58bd2042.png)
